benchmark/jsonresultset/jsonresultset_test.go

```diff
@@ -133,3 +92,22 @@ func runJSONResultSet() {
 		counter++
 	}
 }
+
+func setCustomConfig() {
```

Comment:
I'm not sure if naming a function in so generic manner is a good thing. Maybe split it into two functions like `setCustomJsonDecoder` and `setMaxChunkDownloadWorkers`?

Reply:
Good idea. Done.